src/ansible_compat/runtime.py

```diff
@@ -354,69 +354,69 @@ def _prepare_ansible_paths(self) -> None:
             'ANSIBLE_ROLES_PATH', roles_path, default=ANSIBLE_DEFAULT_ROLES_PATH
         )
 
+    # pylint: disable=no-self-use
```

Comment:
Instead of this, you should probably use `self.project_dir` somewhere in code.

Reply:
This was there only because I wanted to split a bigger change in two for clarity. The next commit did remove it. It was not meant to stay in for long.